repo/packages/K/kafka/38/package.json

```diff
@@ -5,6 +5,7 @@
   "minDcosReleaseVersion": "1.7",
   "name": "kafka",
   "packagingVersion": "3.0",
+  "preInstallNotes": "This DC/OS Service is currently in preview.",
```

Comment:
This should be updated in the `dcos-commons/frameworks/kafka/universe` location, as well.